feat  : MongoDB 모듈 추가

beetrbgus opened the pull request:
## 📃 관련 이슈
- #54 

## ✨ 변경 내용
- 공연장 좌석은 공연 1개에 N개의 회차가 있고, 공연장마다 M개의 좌석이 있기 때문에 굉장히 많아짐
- 예를 들어 공연 1개 당 500개의 회차가 있고, 공연장에는 1000개의 좌석이 있다고 가정하면 한 공연당 500,000개의 좌석이 생기게 됨
- MySQL로 적재시 병목이 생길 것이라고 판단해 MongoDB로 한꺼번에 넣기 위해 MongoDB 모듈을 추가 함.
- 또 MySQL, MongoDB 간의 규격의 불일치로 Domain을 두고 각 infra 모듈에서는 Domain에서 Entity 모양을 받아서 저장

### TODO
- 기본만 했기 때문에 ticket-seat 에서 중복 검사를 해야 함.
## 🔎 참고 사항 (선택)
- 추가 설명, 참고 링크 등

beetrbgus committed:
feat : Venue 관련 domain 수정

# #############################
# # 타입 설정                  ##
# #############################

# # feat: 기능 구현
# # refactor: 코드 구조 개선, 변수명 변경 등 코드 리팩토링
# # fix: 버그 또는 오류 해결
# # style: CSS 등 스타일 관련 변경
# # test: 테스트 코드 추가 또는 수정
# # docs: README.md 등 문서 관련 변경
# # chore: 파일 또는 디렉토리 구조 변경, 패키지 설치 등 빌드 시스템이나 외부 도구 관련 작업
# # build: CI/CD 관련 변경

beetrbgus committed:
feat : 패키지 수정

# #############################
# # 타입 설정                  ##
# #############################

# # feat: 기능 구현
# # refactor: 코드 구조 개선, 변수명 변경 등 코드 리팩토링
# # fix: 버그 또는 오류 해결
# # style: CSS 등 스타일 관련 변경
# # test: 테스트 코드 추가 또는 수정
# # docs: README.md 등 문서 관련 변경
# # chore: 파일 또는 디렉토리 구조 변경, 패키지 설치 등 빌드 시스템이나 외부 도구 관련 작업
# # build: CI/CD 관련 변경

beetrbgus committed:
feat : 패키지 수정

# #############################
# # 타입 설정                  ##
# #############################

# # feat: 기능 구현
# # refactor: 코드 구조 개선, 변수명 변경 등 코드 리팩토링
# # fix: 버그 또는 오류 해결
# # style: CSS 등 스타일 관련 변경
# # test: 테스트 코드 추가 또는 수정
# # docs: README.md 등 문서 관련 변경
# # chore: 파일 또는 디렉토리 구조 변경, 패키지 설치 등 빌드 시스템이나 외부 도구 관련 작업
# # build: CI/CD 관련 변경

beetrbgus committed:
feat : MongoDB 모듈 추가 및 기본 추가

beetrbgus committed:
feat : MongoDB 모듈 추가 및 기본 추가

beetrbgus committed:
Merge branch 'develop' into feat/54

beetrbgus committed:
feat : MongoDB 모듈 추가 및 기본 추가

beetrbgus committed:
feat : 불필요 RepositoryImpl 삭제

beetrbgus commented on showhive-domain/domain/src/main/java/com/showhive/category/domain/CategoryDomain.java:
```diff
@@ -0,0 +1,37 @@
+@Builder
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor(access = AccessLevel.PROTECTED)
+public class CategoryDomain extends BaseEntity {
```
BaseEntity 제거 하는거

beetrbgus committed:
Delete showhive-infra/infra-ticketing-seat/gradle/wrapper/gradle-wrapper.properties

Jiihyun commented on showhive-domain/domain/src/main/java/com/showhive/performance/domain/PerformanceDomain.java:
```diff
@@ -0,0 +1,36 @@
+@Builder
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor(access = AccessLevel.PROTECTED)
+public class PerformanceDomain extends BaseEntity {
```
R: BaseEntity 제거!

Jiihyun commented on showhive-domain/domain/src/main/java/com/showhive/venue/domain/VenueDomain.java:
```diff
@@ -0,0 +1,29 @@
+@Builder
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor(access = AccessLevel.PROTECTED)
+public class VenueDomain extends BaseEntity {
```
R: BaseEntity 제거 필요!

Jiihyun reviewed (commented):
고생하셨슴니다!!

beetrbgus committed:
feat : Domain에 BaseEntity 제거

# feat: 기능 구현
# refactor: 코드 구조 개선, 변수명 변경 등 코드 리팩토링
# fix: 버그 또는 오류 해결
# style: CSS 등 스타일 관련 변경
# test: 테스트 코드 추가 또는 수정
# docs: README.md 등 문서 관련 변경
# chore: 파일 또는 디렉토리 구조 변경, 패키지 설치 등 빌드 시스템이나 외부 도구 관련 작업
# build: CI/CD 관련 변경

Jiihyun reviewed (approved):
수고하셨슴당
